projects/spdlog/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build project
+cmake . && make -j$(nproc)
+
+find . -name "*.a"
+find . -name "spdlog.h"
+
+# build fuzzers
+$CXX $CXXFLAGS -std=c++11 -Iinclude \
+    $SRC/log_fuzzer.cc $LIB_FUZZING_ENGINE ./libspdlog.a \
+    -o $OUT/log_fuzzer
+
+zip $OUT/seed_corpus.zip *.*
```

Comment:
why add every file to the seed corpus?

Reply:
Should I get rid of it if there's no corpus in spdlog yet?

Reply:
Yes

Reply:
I got rid of it. At this spdlog doesn't have corpuses.